cypress/e2e/column-text-link.cy.js

```diff
@@ -0,0 +1,69 @@
+let localUser
+
+describe('Test column text-link', () => {
+
+	before(function() {
+		cy.createRandomUser().then(user => {
+			localUser = user
+			cy.login(localUser)
+			cy.uploadFile('photo-test-1.jpeg', 'image/jpeg', '/photo-test-1.jpeg')
+			cy.uploadFile('NC_server_test.pdf', 'application/pdf', '/NC_server_test.pdf')
+		})
+	})
+
+	beforeEach(function() {
+		cy.login(localUser)
+		cy.visit('apps/tables')
+	})
+
+	it('Create', () => {
```

Comment:
Something I forgot during the call to mention, but non blocking from my perspective. Tests should not depend on each other so that in theory you could run them in random order or independently and makes them retryable. In this spec file every test creates the requirements for the next ones. This also has the benefit that you can run them separately with .only for debugging